task1/UserFile.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace UserFile
+{
+    public class t
+    {
+        public static string Solution(string str)
+        {
+            string sortedStr = f.f.SortString(str);
+            string resultStr = "";
+            int index = 0;
+            List<char> charList = new List<char>();
```

Comment:
```suggestion
            List<char> charList = new();
```
Не указывайте в одном объявлении один и тот же тип дважды.